<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_client</class>
 <widget class="QWidget" name="tcp_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcp_client</string>
  </property>
  <widget class="QPushButton" name="move_to_pos">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>移动到</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_register_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>读寄存器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_server_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>209</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>210</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>机器人：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="robot_box"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>124</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="modbus_tcp_rbtn">
      <property name="text">
       <string>Modbus_tcp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="kawasaki_json_rbtn">
      <property name="text">
       <string>kawasaki_json</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>187</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>机器人位置：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Y：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Z：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="z_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>RX：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rx_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>RY：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ry_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>RZ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rz_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>SP：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sp_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="record_tb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>491</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="laser_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>打开激光</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="send_order_edit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>296</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>任务号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="task_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="write_task_btn">
      <property name="text">
       <string>写入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>491</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="get_pos_btn">
      <property name="text">
       <string>获取位置信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pos_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>491</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QPushButton" name="get_weld_btn">
      <property name="text">
       <string>获取焊缝信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="weld_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
